File: Mega_Project_List_Solutions/Numbers/Fibonacci_Sequence.cpp
#include <iostream>
using namespace std;

int main()
{
	int a = 0, b = 1, c = 0;
	int fib_num = 0, count = 0;
	std::cout << "Enter the place of the fibonacci number you want to find: ";
	std::cin >> fib_num;

	while (count < fib_num)
	{
		count++;
		c = a + b;
		std::cout << c << std::endl; // Remove this line to only see final number.
		a = b;
		b = c;
	}
	std::cout << "The value of the " << fib_num << " place in the Fibonacci Sequence is: " << c << "\n\n";
	return 0;
}

File: Mega_Project_List_Solutions/Numbers/Fibonacci_Sequence.py
a,b,c = 0,1,0
place = int(input("Enter the place of the Fibonacci number you want to find: "))
for x in range(place):
    #print(a) # -- Print the current Fibonacci number.
    c = a
    a += b
    b = c
print("Your request sequence place, {0}, in the Fibonacci Sequence is: {1}".format(place,c))
